# Example of a sensor data message
builtin_interfaces/Time stamp
float32 temperature
float32 humidity
